### PR TITLE
feat(Recruiters): form recruiters

### PR DESCRIPTION
**Pull request by YeLnatSs**
Recruiters form created
Image added on banner

### CODE REVIEW

File: src/i18n/source/pages.yaml
Comment (abruere) [inline src/i18n/source/pages.yaml]: I would move this to https://github.com/abruere/workingoo/blob/dev/src/i18n/theme/pages.yaml to ease updates and sync from upstream project in `i18n/source` files.

File: src/i18n/source/prompt.yaml
Comment (abruere) [inline src/i18n/source/prompt.yaml]: Same.

File: src/layouts/HeroTwoColLayout.vue
Comment (abruere) [inline src/layouts/HeroTwoColLayout.vue]: I suggest you reuse existing `BasicHeroLayout.vue` since there is a slot to easily customize content.

https://github.com/abruere/workingoo/blob/e89998f6ed2ecfd52237da702b2eb2d25d304fe3/src/layouts/BasicHeroLayout.vue#L3-L8

You can add an outer slot in `BasicHeroLayout.vue` if you need more customization

```vue
<section class="hero hero__background-container"> 
   <div class="hero__background bg-primary-gradient" /> 
   <slot name="heroContentContainer">
     <!-- This is default content of heroContentContainer slot -->
     <div class="hero__content text-center text-white"> 
       <slot name="heroContent" /> 
     </div> 
  </slot>
</section> 
```

Note: you can only use _either_ `heroContent` or `heroContentContainer`, it’s all what you need to fully customize the content and keep current hero background.

File: src/pages/Recruiters.vue
Comment (abruere) [inline src/pages/Recruiters.vue]: We won’t be able to maintain duplicates of pages.

I suggest you use the same `NewAsset.vue` page you can also restyle if needed (depending on `this.$route`)

File: src/router/routes.js
Comment (abruere) [inline src/router/routes.js:52]: Here you can also map to newAsset, and detect `this.$route.name` in this page to know if it `newAsset` or `recruiters`, to reuse the same component and avoid duplicating code.

```
{
      name: 'recruiters',
      path: 'r',
      component: () => import(/* webpackChunkName: 'landing' */ 'pages/NewAsset.vue'),
      meta: {}
    },
```

**Review by abruere — COMMENTED** [review]
Thanks for contributing!

As explained below we need to reuse existing code as much as possible to prevent bugs in the future.

The page looks very much like current `NewAsset`, except for the hero that can be customized. Same for input style.

https://deploy-preview-1--stelace-instant-workingoo-test.netlify.com/r
https://deploy-preview-1--stelace-instant-workingoo-test.netlify.com/n


Thanks to this PR I also spotted a responsive bug in `NewAsset.vue` with `title` input I’ll fix from upstream jobs project.

**Comment by abruere** [comment]
I’ve fixed the input and made changes to `BasicHeroLayout.vue` I suggested above https://github.com/stelace/jobs-marketplace-template/commit/0ca95c981b14cddf8b2e7ff9ff48f33b351371fe to ease your work. 

@YeLnatSs you can rebase on dev branch to use this updated component instead of `HeroTwoColLayout.vue`, and get other latest updates in your fork.

```
git checkout dev
git fetch upstream
git merge upstream/dev
git checkout form-recruiters
git rebase dev
```

File: src/i18n/source/pages.yaml
Comment (YeLnatSs) [reply src/i18n/source/pages.yaml]: Okay, I moved it to the other location.

File: src/i18n/source/prompt.yaml
Comment (YeLnatSs) [reply src/i18n/source/prompt.yaml]: This one too.

File: src/layouts/HeroTwoColLayout.vue
Comment (YeLnatSs) [reply src/layouts/HeroTwoColLayout.vue]: I worked from your changes and reuse `BasicHeroLayout.vue`. Based on `this.$route` element is displayed.

File: src/pages/Recruiters.vue
Comment (YeLnatSs) [reply src/pages/Recruiters.vue]: No problem. Thank you

File: src/router/routes.js
Comment (YeLnatSs) [reply src/router/routes.js:52]: I did Thank you

File: src/pages/NewAsset.vue
Comment (abruere) [inline src/pages/NewAsset.vue:158]: Are sure you want to get rid of several steps revealed as user fills the form?

Antoine specifically asked for this initially but he may have changed his mind.

You can disable this behavior only when `isRecruiters === true` if you want to keep steps for other users.

```js
  step () {
    const steps = [
      true, // fictive step 0
      true,
      false,
      false,
      false,
    ]
    if (isRecruiters) return steps.map(s => true)

    if (this.name.length >= 1) { // with high debounce to reduce distraction while typing
      steps[2] = true
    }
    …
```

**Review by abruere — COMMENTED** [review]
Thanks for the changes and reusing existing components.

Please find a question about NewAsset form below.

If you’re satisfied with the preview (link in netlify CD details below), you can rebase your branch and let me know so I can merge into dev branch.

File: src/pages/NewAsset.vue
Comment (YeLnatSs) [reply src/pages/NewAsset.vue:158]: Miguel told me to remove the effect that reveals the fields once filled because it was causing problems and for UX design.

**Comment by YeLnatSs** [comment]
@abruere It seems good to me. I rebased my branch

**Review by abruere — CHANGES_REQUESTED** [review]
Hi @YeLnatSs, this PR currently breaks header:

deploy-preview-1--stelace-instant-workingoo-test netlify com

File: src/layouts/MainLayoutHeader.vue
Comment (abruere) [inline src/layouts/MainLayoutHeader.vue:370]: Broken header is probably due to your `v-show` being on the child of the div you added, rather than on parent:

```js
<div class="q-mx-md col-md-9">
        <div
          v-show="isRecruiters"
          :class="{ 'recruiters-title': isRecruiters }"
          class="text-warning text-weight-medium text-center text-uppercase text-subtitle1"
        >
        {{ $t({ id: 'pages.recruiters.tagline' }) }}
       </div>
</div>
```

to 

```js
<div v-show="isRecruiters" class="q-mx-md col-md-9">
        <div
          :class="{ 'recruiters-title': isRecruiters }"
          class="text-warning text-weight-medium text-center text-uppercase text-subtitle1"
        >
        {{ $t({ id: 'pages.recruiters.tagline' }) }}
       </div>
</div>
```

Comment (YeLnatSs) [reply src/layouts/MainLayoutHeader.vue:370]: Okay, I changed it. I'm waiting for a feedback before commit it

File: src/i18n/source/asset.yaml
Comment (abruere) [inline src/i18n/source/asset.yaml]: typo: `jobs`

File: src/i18n/theme/pricing.yaml
Comment (abruere) [inline src/i18n/theme/pricing.yaml]: EDIT: ~If changed in `en` it also needs to be changed to `Salaire net` in `fr`.~

Done in next commit thanks :)

**Review by abruere — APPROVED** [review]
Looks ready for merge once you’ve resolved conflicts due to recent updates.

I also suggest you add some padding to `Tout commence par un bon titre` line before first input on /n reworked page.

_Note: I let you check with Workingoo that `/r` URL for this new page is ok._

**Comment by YeLnatSs** [comment]
> Looks ready for merge once you’ve resolved conflicts due to recent updates.
> 
> I also suggest you add some padding to `Tout commence par un bon titre` line before first input on /n reworked page.
> 
> _Note: I let you check with Workingoo that `/r` URL for this new page is ok._

Thank. I fixed conflicts and I added some padding on `Tout commence par un bon titre`

File: src/i18n/source/asset.yaml
Comment (YeLnatSs) [reply src/i18n/source/asset.yaml]: Thanks. I changed it

**Comment by abruere** [comment]
Netlify reports some [linting errors](https://app.netlify.com/sites/stelace-instant-workingoo-test/deploys/5d514d9aaa320700099ecb01). Can you fix them before I merge?

**Comment by YeLnatSs** [comment]
> Netlify reports some [linting errors](https://app.netlify.com/sites/stelace-instant-workingoo-test/deploys/5d514d9aaa320700099ecb01). Can you fix them before I merge?

I fixed them. Normally, you won't have problem to merge.

**Pull request merged**
